buildSrc/src/main/java/com/uber/okbuck/core/dependency/exporter/DependencyExporterModel.java

```diff
@@ -0,0 +1,73 @@
+package com.uber.okbuck.core.dependency.exporter;
+
+import org.codehaus.plexus.util.StringUtils;
+import org.gradle.api.artifacts.ExternalDependency;
+
+import javax.annotation.Nullable;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+public class DependencyExporterModel {
+
+  @Nullable
+  private final String name;
+  @Nullable
+  private final String version;
+  @Nullable
+  private final String group;
+  private final boolean force;
+  @Nullable
+  private Set<String> excludeRules = new TreeSet<>();
+  private final boolean transitive;
```

Comment:
we currently use this property. do we want to export it?

Reply:
removed it